aqueduct_client/utils.py

```diff
@@ -0,0 +1,40 @@
+import os
+import logging
+
+try:
+    # py3
+    from configparser import ConfigParser
+except ImportError:
+    # py2
+    from ConfigParser import SafeConfigParser as ConfigParser
+
+
+def load_api_key():
+    # check in credentials file
+    path = os.path.expanduser("~/.quantopian/credentials")
```

Comment:
I'm not sure what we should do about this, but using `~/.quantopian` will be seen as slightly belligerent by at least some Linux users. There's a somewhat popular standard for managing config like this on linux/bsd: https://specifications.freedesktop.org/basedir-spec/basedir-spec-0.6.html, which says to use `$XDG_CONFIG_HOME` as the config root, with a default value of `~/.config`. There have been a bunch of issues in IPython/Jupyter complaining that they don't follow this:

- https://github.com/ipython/ipython/issues/6070
- https://github.com/jupyter/notebook/issues/2374 (many links to other issues here).

I'm not sure if the convention of `~/.<thing>` or `~/.config/<thing>` is common/appropriate on Windows.

Reply:
I followed AWS's example on this (boto), so I'm comfortable trying this for now and iterating as needed.